holoviews/core/data/dask.py

```diff
@@ -229,12 +229,8 @@ def dframe(cls, columns, dimensions):
         return columns.data.compute()
 
     @classmethod
-    def length(cls, dataset):
-        """
-        Length of dask dataframe is unknown, always return 1
-        for performance, use shape to compute dataframe shape.
-        """
-        return 1
+    def nonzero(cls, dataset):
+        return True
 
```

Comment:
This is a definite improvement!  Hardcoding nonzero is vastly better than hardcoding length.  Even so, is there no way to determine the actual value of nonzero in a way that doesn't load the entire dataset?